### ISSUE TITLE
A bit OSX oriented

### ISSUE THREAD

**Issue by kristianbenoit** [hsuanxyz/kbd-generator#9]
A way to handle Keyboards type, PC/MAC. There could have a Keyboard type (PC/MAC) option, and perhaps styling option so that arrows could be ◀, ▲, ▶, ▼ instead of ←, ↑, →, ↓

**Comment by macx** [comment]
Yes, we need this. At the moment, it's a perfect tool for key combos for the computer you are working with, but not a tool to use for documentations and articles I'm writing, if I want to support both. Instead of Live-View, we need an emulator mode as well with windows-keys.
